sdcm/utils/replication_strategy_utils.py

```diff
@@ -30,6 +30,9 @@ def apply(self, node: BaseNode, keyspace: str):
         with node.parent_cluster.cql_connection_patient(node) as session:
             session.execute(cql)
 
+    def get_replication_factors(self) -> list:  # pylint: disable=no-self-use
```

Comment:
could be a property instead `get` method

Reply:
changed